docs/contributors/opentracingprovider.adoc

```diff
@@ -177,6 +177,18 @@ The value should be used to set the componentType field on a `Component` node.
 
 If the transaction has not been defined in the trace context, then the value of this tag will be used to set it. This value will then be propagated to other subsequent trace contexts.
 
+==== Tag ‘service’
+
+A free-format value that ties this request to a specific service. When the application is executed in OpenShift, this defaults to a value that is derived from the environment variable `OPENSHIFT_BUILD_NAME`,
```

Comment:
Shouldn't it also mention the HAWKULAR_APM_SERVICE_NAME as a container independent env variable, and what is precedence is to the OPENSHIFT... vars?